packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -176,4 +179,27 @@ export class FlowBuilderApi {
   private containsAnyKeywords(input: string, keywords: string[]): boolean {
     return keywords.some(keyword => input.includes(keyword))
   }
+
+  getPayloadFromBotActionNodeId(id?: string): string | undefined {
```

Comment:
Why are you creating this method? The `getNodeById` method should do what this method does, no?
Then when having the node, you should access `node.payload` to get the payload.

Reply:
I have refactored and changed a lot this function but what you say I would say that it is not like that if you want tomorrow we will talk and I will explain you how it is done.

Reply:
Yes, I was confused, sorry for the noise